README.md

```diff
@@ -1 +1,6 @@
-# Hello-Microverse-Project
\ No newline at end of file
+# Hello-Microverse-Project
+
+Title : Hello Microverse Project
+Purpose: The project is meant for learning how to use git and git.
+It aims to ensure that the following is achieved at the end of the session:
+1.Create branches with different base branches. 2. Understand the concept of version control. 3. Follow GitHub flow. 4. Open pull requests in GitHub. 5. Write descriptive, easy to understand commit messages. 6. Write short, detailed, easy to understand descriptions of pull requests. 7. Write a descriptive and easy to understand README in business English for every project submitted. 8. Demonstrate an ability to submit a project for code review.
```

Comment:
- [ ] Kindly use the [template provided by Microverse](https://github.com/microverseinc/readme-template) to document your code. Remember that having professional code documentation will make your projects more attractive to people that want to collaborate on them and also will provide better guidance on how to interact with them. Please consider using the template and customizing it with the project information! 🤓 🛠️